backend/src/ee/services/secret-approval-request/secret-approval-request-secret-dal.ts

```diff
@@ -11,6 +17,27 @@ export const secretApprovalRequestSecretDALFactory = (db: TDbClient) => {
   const secretApprovalRequestSecretOrm = ormify(db, TableName.SecretApprovalRequestSecret);
   const secretApprovalRequestSecretTagOrm = ormify(db, TableName.SecretApprovalRequestSecretTag);
 
+  const bulkUpdateNoVersionIncrement = async (
+    data: Array<{ filter: Partial<TSecretApprovalRequestsSecrets>; data: TSecretApprovalRequestsSecretsUpdate }>,
+    tx?: Knex
+  ) => {
+    try {
+      const secs = await Promise.all(
+        data.map(async ({ filter, data: updateData }) => {
```

Comment:
We are actually only getting 100 versions per folder, the rest are discarded.

If I was going to use the `insert` method you mentioned, I'd need a full object instead of just a partial update object.

Do you think this will still be an issue, considering we're only updating 100 versions per folder?

Reply:
100 is still in my opinion could be a bit trouble someone. If it was insert i mean 1000 wouldn't have matter. Where as 100 its individual query. N query this is not one. Where as the other one is 1 single query